programmers/난이도별/level01.제일_작은_수_제거하기/SSSOy.java

```diff
@@ -0,0 +1,26 @@
+import java.util.*;
+
+class Solution {
+    public int[] solution(int[] arr) {
+        
+        int min = arr[0];
+        List<Integer> list = new ArrayList<Integer>();
+        for (int i : arr) {
+            list.add(i);
+            if(min > i) 
+                min = i;
+        }
+        
+        list.remove((Integer)min);
+        
+        int[] answer = new int[list.size() > 0? list.size() : 1];
+        int size = 0;
+
+        for(int temp : list)
+            answer[size++] = temp;
+        if(size == 0)
+            answer[0] = -1;
+
+        return answer;
```

Comment:
앞에서 최솟값을 구했다면, 다음과 같이 바로 리턴하면 된답니다!
```suggestion
        return Arrays.stream(arr)
                     .boxed()
                     .filter(v -> v != min)
                     .mapToInt(v -> v)
                     .toArray();
```

Reply:
스트림을 사용하면 이렇게 간단하게 풀 수 있군요..!!
좋은 리뷰 감사합니다~😊